abc.templates/base-workflows/contents/workflows/call-guardian-plan.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Call Guardian Plan is used to call Reusable Guardian Plan workflow
+name: 'call_guardian_plan'
+on:
+  pull_request:
```

Comment:
Should this be `pull_request_target`?

Reply:
FYI, I could be completely wrong here... I'm trying to remember how we wanted this to work.

Reply:
i dont think, the goal was to get rid of pull_request_target. we need the plan run to come from main, not necessarily this file

Reply:
I was wrong. `pull_request` is correct here. `pull_request` is inherently safer because it does has read-only access to the repo and no access to secrets (as opposed to `pull_request_target`). So we want to use `pull_request` whenever possible.